animareflection/ui
# Example App Components

## Description

##### Task link: https://trello.com/c/ualZD8HW/149-add-components-to-example-repos

Added components to the example app repos. During the process, found the need to update `Accordion`, nextjs enhanced `Image`, and `Slider` components, so proceeded with that as well. The addition of these components will make future PR reviews much easier, as the plan is to be adding / confirming the components work in "downstream apps" through the example repos.

## Test Steps

1) Verify that the updates to the components make sense.
2) Verify that the components render appropriately in downstream (example) apps.
